code/style.css

```diff
@@ -37,23 +40,48 @@ input {
   box-sizing: border-box;
   border: none;
   border-radius: 4px 0 0 4px;
-  background: #e5e9ff;
-  color: #0026ff;
+  background: #eba059;
+  color: #f6f6f6;
   padding: 16px;
   font-size: 16px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat";
   font-weight: 600;
   line-height: 26px;
   flex: 1;
 }
 
+input:focus,
+select:focus {
+  outline: none;
```

Comment:
It often looks nicer without the outline, for sure, but there are people with disabilities, who can't use standard computers with mouse or touchpad. Maybe it is a better way to make your outline more delicate, and elegant, but still visible for people who needs it to navigate through your page?